lib/tasks/impossible_overdue_appointments.rake

```diff
@@ -0,0 +1,47 @@
+namespace :impossible_overdue_appointments do
```

Comment:
The namespace and filename can be more generic. `data_fixes.rake` maybe?

Reply:
Updated namespace and filename to `data_fixes`.